defaults/main.yml

```diff
@@ -1,13 +1,12 @@
----
-bigdft_version: "1.9.0"
-bigdft_src: bigdft-suite-{{ bigdft_version }}
-bigdft_src_archive: "{{ bigdft_src }}.tar.gz"
+bigdft_version: "4855f2cd" # "1.9.0"
 bigdft_code_folder: "/tmp"
 bigdft_prefix: "/usr/local"
 bigdft_python_path: "{{ ansible_python_interpreter | default('/usr/bin/python') }}"
-bigdft_url: "https://gitlab.com/l_sim/bigdft-suite/-/archive/{{ bigdft_version }}/{{ bigdft_src_archive }}"
 bigdft_topdir: "{{ bigdft_code_folder }}/bigdft-suite"
+# bigdft_src: bigdft-suite-{{ bigdft_version }}
```

Comment:
FYI @adegomme I also commented all these out, since we are not actually using them

Reply:
yes, it's still temporary while we use a git version, so I will activate it back when needed.